Assets/Content/Items/Cosmetic/BikeHorn.cs

```diff
@@ -74,26 +73,37 @@ public void Cancel(InteractionEvent interactionEvent, InteractionReference refer
         public override void Start()
         {
             base.Start();
-            audioSource = GetComponent<AudioSource>();
-            GenerateNewIcon(); 
+            GenerateNewIcon();
         }
 
         private bool IsHonking()
         {
-            return audioSource.isPlaying;
+            //If we have an audio source and it's playing, return true. Otherwise, return false. (A new audio source will be claimed.)
+            if (audioSource != null)
+            {
+                return audioSource.isPlaying;  
```

Comment:
The audio source could be reused after it has finished. If it is then used to play a longer sound, you will be unable to honk :(

Reply:
I was considering that, keeping track of whether or not the audio source is playing kinda sucks here, since there's no specific audio source. Any ideas on how to make that better? I think there's probably some need for a noisemaker script for things like this, and the service bell, that just use timed cooldowns rather than keeping track of whether the audio is being played.